tools/pd-api-bench/cases/cases.go

```diff
@@ -164,6 +167,11 @@ func (c *minResolvedTS) Do(ctx context.Context, cli *http.Client) error {
 	return nil
 }
 
+func (c *minResolvedTS) Params(param string) {
```

Comment:
add in `api-bench`
after that we can run `go run main.go -http-cases GetMinResolvedTS-1+1 -params 'scope=cluster'  -client 1 -debug  `
@CabinfeverB @lhy1024  PTAL, thx!
![image](https://github.com/tikv/pd/assets/53859786/e5d20fcd-1a2f-4f87-bb33-52994a2b335b)

Reply:
Does it support grpc too?

Reply:
this is for api's params, I think it's no need to add in grpc for now